DigitalLearningSolutions.Web/Views/SsoLinkage/Index.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@{
+  ViewData["Title"] = "Single Sign On Linkage";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-u-maring-bottom-8">
+      <h1 class="nhsuk-heading-x1">You need to link your account</h1>
+    </div>
+    <div class="nhsuk-contents-list">
+      <p class="nhsuk-body-m">In the future we will be managing you log in to DLS and the Learning hub using a single account. Linking your account to the Learning Hub now should take no longer than 2 minutes.</p>
+      <p>Linking your account will:
+        <ul>
+          <li>Enable you to access Learning Hub content from DLS saving you time,</li>
+          <li>give you access to thousands of learnin gresource through the Learning Hub,</li>
+          <li>and, accessing learning content on DLS and the learning hub will be quicker and easier</li>
+        </ul>
+      </p>
+    </div>
+    <div>

```

Comment:
Subject to Allan confirming, I think we should include something in here about elfh Hub accounts being the same as LH accounts. Perhaps NHS Design System inset text:

![image](https://github.com/TechnologyEnhancedLearning/DLSV2/assets/67740339/33e60193-d8b5-4364-ae6b-4f0ea6c693e3)